.github/workflows/release.yml

```diff
@@ -29,7 +29,6 @@ jobs:
       publish_docs: ${{ steps.check_commits.outputs.publish_docs }}
       commit_details: ${{ steps.check_commits.outputs.commit_details }}
       append_notes: ${{ steps.check_commits.outputs.append_notes }}
-      pr_description: ${{ steps.get_pr_description.outputs.pr_description }}
     steps:
       - uses: actions/checkout@v3
```

Comment:
- In the `create-confluence-page` job, some comments mention "PR description" which is no longer part of the workflow. Remove any misleading references.